spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/main/java/org/springframework/cloud/stream/binder/kafka/KafkaProducerProperties.java

```diff
@@ -41,6 +43,14 @@ public void setBufferSize(int bufferSize) {
 		this.bufferSize = bufferSize;
 	}
 
+	public int getMaxRequestSize() {
```

Comment:
The definitely makes sense. I was attempting to follow bufferSize as a pattern but I'm not seeing that parameter referenced outside this source file - just assumed there was Spring magic in play here. Any pointers you can provide would be helpful and I'll try to follow up with the rest of the implementation.